uktrade/data-hub-frontend
# Reactify investment associated project page

## Description of change

The page for linking an associated R&D project with an investment project has been reactified. The logic for editing and removing these projects has also been reactified.

## Test instructions

Go to an investment project. You should be able to select an associated project, change the project and remove it (only if the project isn't at the `Verify win` or `Won` stages).

## Screenshots

![Screenshot 2023-08-30 at 13-37-55 Update associated project - New airport - Projects - Investments - DBT Data Hub](https://github.com/uktrade/data-hub-frontend/assets/36161814/de67cca8-a03d-4b43-9629-8c54986f19d5)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
